.travis.yml

```diff
@@ -333,21 +341,21 @@ deploy:
    endpoint: s3-us-west-2.amazonaws.com
    acl: public_read
    on:
-     condition: $TRAVIS_OS_NAME = osx && ! -z $AWS_KEY && ! -z $AWS_SECRET && ! -z $DO_DEPLOY
+     condition: $TRAVIS_OS_NAME = osx && ! -z $AWS_KEY && ! -z $AWS_SECRET && ! -z $DEPLOY_AWS_PREFIX
      all_branches: true
  # github releases - only tags
  - provider: releases
    api_key: $GITHUB_KEY
-   file: $HOME/SuperCollider-$VERSION_TO_BUILD-macOS.zip
+   file: $HOME/$ARTIFACT_GH_NAME.zip
    prerelease: true
    skip_cleanup: true
    on:
-     condition: $TRAVIS_OS_NAME = osx && ! -z $GITHUB_KEY && ! -z $DO_DEPLOY
+     condition: $TRAVIS_OS_NAME = osx && ! -z $GITHUB_KEY && ! -z $DEPLOY_GITHUB_SUFFIX
      tags: true
      all_branches: true
 
 after_deploy:
- - "echo S3 Build Location: $S3_URL"
+ - 'if [[ ! -z "$DEPLOY_AWS_PREFIX" ]]; then echo S3 Build Location: $S3_URL; fi' 
```

Comment:
don't think you need the single quotes? and can simplify to `-n` instead of `! -z`. you can do that throughout this script if you want in fact

Reply:
I think I was getting errors without the quotes, so left them.